### ISSUE TITLE
SLE-12 is LTSS now, rules need to be updated to reflect this

### ISSUE THREAD

**Issue by plusky** [openSUSE/repose#5]

**Comment by plusky** [comment]
Already resolved by 9a65477b7d5b9322defe8c77a6187592cd65d586

**Comment by mimi1vx** [comment]
```
repoq -A -a x86_64 -t gm -t up  -t se  -t lt sles:12
sles:12::gm zypper -n ar -cgkn sles:12::gm http://download.suse.de/ibs/SUSE/Products/SLE-SERVER/12/x86_64/product/ sles:12::gm
sles:12::up zypper -n ar -cgkfn sles:12::up http://download.suse.de/ibs/SUSE/Updates/SLE-SERVER/12/x86_64/update/ sles:12::up
```

mentioned commit only resolves problem with 11SPx LTS repos
